docs/root/configuration/other_protocols/thrift_filters/router_filter.rst

```diff
@@ -41,3 +41,5 @@ Since these stats utilize the underlying cluster scope, we prefix with the ``thr
   thrift.upstream_resp_exception, Counter, Total responses with the "Exception" message type.
   thrift.upstream_resp_invalid_type, Counter, Total responses with an unsupported message type.
   thrift.upstream_rq_time, Histogram, total rq time from rq complete to resp complete; includes oneway messages.
+  thrift.upstream_rq_size, Histogram, Request message size in bytes per upstream
+  thrift.upstream_rs_size, Histogram, Response message size in bytes per upstream
```

Comment:
What do you think about s/rs/resp/ to make it match the other response metrics?

Reply:
Yeah I'd agree, the HTTP ones (see  #15908) go with `rs` – but I don't think consistency across HTTP and Thrift stats matters that much.

Reply:
Done.